.github/workflows/phpunit.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres
+        image: postgres:14
```

Comment:
Mind to also adjust in the integration tests right away? :)

Reply:
:+1: